decentraland/AppNavigator.js

```diff
@@ -7,9 +15,27 @@ import OnboardingHomeScreen from "./screens/OnboardingHomeScreen";
 import EthereumQuestionScreen from "./screens/EthereumQuestionScreen";
 import EthereumSignUpScreen from "./screens/EthereumSignUpScreen";
 import EthereumSignInScreen from "./screens/EthereumSignInScreen";
+import MyAssetsScreen from "./screens/MyAssetsScreen";
 import Colors from "@constants/Colors.js";
 
-const AppNavigator = createStackNavigator(
+const defaultNavigationOptions = ({ navigation }) => {
+  return {
+    headerLeft: (
+      <Button transparent onPress={() => navigation.toggleDrawer()}>
+        <Icon name="menu" />
+      </Button>
+    ),
+    headerStyle: {
+      backgroundColor: Colors.headerBackgroundColor,
```

Comment:
I think we need a header color that's less dramatically different from the main background color (or perhaps even the same color)

Reply:
I've changed to gray. What do you think?

Reply:
The more I think about it, I'm personally leaning towards a transparent header that's the exact same color as the background. I'll see how the gray looks!

Reply:
This won't gate merging but I'm still curious for your thoughts

Reply:
I'll see if white looks like. Refs: https://github.com/tasitlabs/tasit/issues/201

Reply:
Great, thanks. We can move the discussion there and resolve this. Resolving it now.